File: fe/src/app/models/note.ts
export interface INote {
  id?: number;
  postId: number;
  content: string;
  createdDate?: number;
  userId?: string;
  userName?: string;
}

export class Note implements INote {
  id?: number;
  postId: number;
  content: string;
  createdDate?: number = 0;
  userId?: string;
  userName?: string;

  constructor(init?: INote) {
    if (init) {
      this.id = init.id || 0;
      this.postId = init.postId;
      this.content = init.content;
      this.createdDate = init.createdDate || 0;
      this.userId = init.userId;
      this.userName = init.userName;
    } else {
      this.id = 0;
      this.postId = 0;
      this.content = '';
    }
  }
}

File: Domain/Note.cs
namespace Domain
{
    public class Note
    {
        public int Id { get; set; }
        public int PostId { get; set; }
        public long CreatedDate { get; set; }
        public Guid UserId { get; set; }
        public string UserName { get; set; }
        public string Content { get; set; }
    }
}